data/processors/export.py

```diff
@@ -115,6 +115,8 @@ def export_for_api(data, path):
             del export_data[_id]["tumonline_data"]
         if "roomfinder_data" in export_data[_id]:
             del export_data[_id]["roomfinder_data"]
+        if "nat_data" in export_data[_id]:
+            del export_data[_id]["nat_data"]
```

Comment:
given that these are now 3, I would prefer (no strong oppinion)
```suggestion
        for key in ["tumonline_data", "roomfinder_data", "nat_data"]
            if "nat_data" in export_data[_id]:
                del export_data[_id][key]
```